app/controllers/api_controller.rb

```diff
@@ -0,0 +1,17 @@
+class ApiController < ApplicationController
```

Comment:
no no no)
you need to have `controller/api/` -> 
1 `ApiController < ApplicationController`
2 `MicropostsController < ApiController`